examples/nine-pt-stencil-solver/doc/intro.dox

```diff
@@ -4,36 +4,36 @@
 This example solves a 2D Poisson equation:
 
 $
-    \Omega = (0,1)^2
-    \Omega_b = [0,1]^2   (with boundary)
-    \partial\Omega = \Omega_b \backslash \Omega
-    u : \Omega_b -> R
-    u'' = f in \Omega
-    u = u_D on \partial\Omega
+    \Omega = (0,1)^2 \\
+    \Omega_b = [0,1]^2   (with boundary) \\
```

Comment:
I think you are still missing the `\text{}` around the `with boundary`:
```suggestion
    \Omega_b = [0,1]^2 \text{  (with boundary)} \\
```
Otherwise, it expects each letter to be a variable (therefore, ignoring every spacing). That makes it really hard to read (see [the current example page](https://ginkgo-project.github.io/ginkgo/doc/develop/nine_pt_stencil_solver.html))